e2e/tests/settingsConfig.spec.ts

```
@@ -17,112 +17,217 @@ test.describe('Settings Tests', () => {
+        expect(value === Data.roleModificationData[7].input).toBe(true);
+        await apiCall.modifySettingsRole(roles.cmdInfo, "yes")
+    });
+
+    test(`@admin Modify maxInfoQueries via UI validation via API: Input value: ${Data.roleModificationData[8].input}`, async () => {
+        const settingsConfigPage = await browser.createNewPage(SettingsConfigPage, urls.settingsUrl)
+        await settingsConfigPage.modifyRoleValue(roles.maxInfoQueries, Data.roleModificationData[8].input)
+        await settingsConfigPage.refreshPage();
+        await settingsConfigPage.scrollToBottomInTable();
+        await settingsConfigPage.getRoleContentValue(roles.maxInfoQueries);
+        const apiCall = new ApiCalls()
+        let value = String((await apiCall.getSettingsRoleValue(roles.maxInfoQueries)).config[1]);
```

ESLint

[error] 'value' is never reassigned. Use 'const' instead.

(prefer-const)